Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/PokePointer.cs

```diff
@@ -171,7 +171,7 @@ private bool FindClosestTouchableForLayerMask(LayerMask layerMask, out BaseNearI
 
             for (int i = 0; i < numColliders; ++i)
             {
-                var touchable = queryBuffer[i].GetComponent<ColliderNearInteractionTouchable>();
+                var touchable = queryBuffer[i].GetComponent<BaseNearInteractionTouchable>();
```

Comment:
@LukasTeonneMS, I think this should be fine because the UnitUI touchables will not be hit by the spherecast anyway

Reply:
But why is it necessary?

Reply:
Per request from @julenka, I refactored the class hierarchy.  ColliderNearInteractionTouchable has been deleted.  See this whiteboard drawing of the old vs new hierarchy.  (NIT = NearInteractionTouchable)

![mrtk_nearinteraction_class_hierarchy (2)](https://user-images.githubusercontent.com/41760870/64274755-e3c41300-cef8-11e9-9295-4f773f41482c.jpg)

The red INIT was an interface I had created in an earlier iteration of this PR to let PressableButton talk with NearInteractionTouchable AND NearInteractionTouchableUnityUI.  Now it talks with them both via NearInteractionTouchableSurface.